examples/ShowPlayer.cpp

```diff
@@ -130,16 +135,21 @@ int ShowPlayer::Playback(unsigned int iterations,
  */
 void ShowPlayer::Loop() {
   ShowLoader::State state = SeekTo(m_start);
-  if (state != ShowLoader::State::OK) {
-    m_next_task = TASK_COMPLETE;
-    if (state == ShowLoader::State::INVALID_LINE) {
-      m_status = ola::EXIT_DATAERR;
-    } else {
+
+  switch (state) {
+    // Success conditions
+    case ShowLoader::END_OF_FILE:
+    case ShowLoader::OK:
       m_status = ola::EXIT_OK;
-    }
-    if (!m_simulate) {
-      m_client.GetSelectServer()->Terminate();
-    }
+      break;
+      // All other states are considered errors of some sort
```

Comment:
I think this should be de-dented to match the word case shouldn't it, as it applies to the following block.